docs/src/suitable_commands.md

```diff
@@ -0,0 +1,55 @@
+# Suitable commands
```

Comment:
To merge it we can:

- Move this document to `working-group` folder 
- Adjust it to reflect all commands we want to support and flag ones that will be done in the first place.

Reply:
`working-group`
is it a naming convention for what we are doing or in progress?

Reply:
It is not docs - just research doc so use whatever name to reflect it